packages/app/src/settings/project/CloudSettings.vue

```diff
@@ -1,5 +1,15 @@
 <template>
-  <ProjectId :gql="props.gql" />
+  <ProjectId
+    v-if="props.gql.currentProject?.projectId"
+    :gql="props.gql"
+  />
```

Comment:
@lmiller1990 Since this component is only rendered here, 
WDYT is re removed the snippet below from `ProjectId`, I guess the assertion `v-if="props.gql.currentProject?.projectId"` is now redundant as result of recent changes
```
<CloudConnectButton
   v-else
   :gql="props.gql"
/>
```

Reply:
Sure, let's try - but the TS compiler may complain.